File: Python/10818.py
import sys

N = int(sys.stdin.readline().strip())
N_list = list(map(int, sys.stdin.readline().strip().split()))

print(min(N_list), max(N_list))
    
"""
과정 및 함수 설명
함수

과정
1. 값을 입력받는다.
2. 최댓값과 최솟값을 출력한다.
"""

File: C/백준/Step4/10818.c
# include <stdio.h>

int main () {
    int N;
    
    scanf("%d", &N);
    int Arr[1000000]; // 실제로는 Arr의 길이에 N을 넣고 풀이하였음
    for (int i = 0; i < N; i++) {
        scanf("%d", &Arr[i]);
    }

    int max = Arr[0];
    int min = Arr[0];
    for (int i = 0; i < N; i++) {
        if (max < Arr[i]) {
            max = Arr[i];
        }
        
        if (min > Arr[i]) {
            min = Arr[i];
        }
    }
    
    printf("%d %d\n", min, max);
    
    return 0;
}